api/src/services/revisions.ts

```diff
@@ -22,4 +22,36 @@ export class RevisionsService extends ItemsService {
 
 		await service.updateOne(revision['item'], revision['data']);
 	}
+
+	private setDefaultOptions(opts?: MutationOptions): MutationOptions {
+		if (!opts) {
+			return { autoPurgeCache: false, bypassLimits: true };
+		}
+
+		if (!('autoPurgeCache' in opts)) {
+			opts.autoPurgeCache = false;
+		}
+
+		if (!('bypassLimits' in opts)) {
+			opts.bypassLimits = true;
+		}
+
+		return opts;
+	}
+
+	override async createOne(data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey> {
+		return super.createOne(data, this.setDefaultOptions(opts));
+	}
+
+	override async createMany(data: Partial<Item>[], opts?: MutationOptions): Promise<PrimaryKey[]> {
+		return super.createMany(data, this.setDefaultOptions(opts));
+	}
+
+	override async updateOne(key: PrimaryKey, data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey> {
+		return super.updateOne(key, data, this.setDefaultOptions(opts));
+	}
+
+	override async updateMany(keys: PrimaryKey[], data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey[]> {
+		return await super.updateMany(keys, data, this.setDefaultOptions(opts));
```

Comment:
```suggestion
		return super.updateMany(keys, data, this.setDefaultOptions(opts));
```

I presume we can remove this, but I might be missing some context 🧐

Reply:
I dont think it technically matters as promises are collapsed when awaiting. However i think i prefer the explicit `await` here 🤔

Reply:
Good catch! 👍 Not sure how the await gotten there but let's remove the await as we are not doing any further processing with the result in this case.